guides/doc-Upgrading_Project/master.adoc

```diff
@@ -1,7 +1,8 @@
 include::common/attributes.adoc[]
 include::common/header.adoc[]
-:upgrade-guide:
-:context: upgrade-guide
+:upgrading-connected:
+:context: upgrading-connected
+:mode: connected
```

Comment:
Does this, by default, break all links because every anchor ends with "_{context}"? If so, we should really focus on changing it **once** and then be happy with it for a long time.

In general, I have no objects to this change per se.

Reply:
@Lennonka @adamlazik1 @mjivraja could you weigh in on this one?

Reply:
I would lean towards this, the way it is in installation guides. (Even though the checks pass).
```
:context: {project-context}
:mode: connected
```
Also, renaming the directory from `doc-Upgrading_Disconnected_Project` to `doc-Upgrading_Project_Disconnected`

Reply:
Hi @Lennonka what do you think should we do here?

Reply:
I think we should have a different context per guide. I'm against using project-context here. Let's keep it as it is.

Reply:
> Also, renaming the directory from `doc-Upgrading_Disconnected_Project` to `doc-Upgrading_Project_Disconnected`

And I agree with Malhar on this.